cfgov/unprocessed/apps/regulations3k/js/index.js

```diff
@@ -1,19 +1,33 @@
-import Expandable from '../../../js/organisms/Expandable.js';
-import { instantiateAll } from '../../../js/modules/util/atomic-helpers';
+import Expandable from 'cf-expandables/src/Expandable';
+import { bindEvent } from '../../../js/modules/util/dom-events';
+import { queryOne as find } from '../../../js/modules/util/dom-traverse';
 
-instantiateAll( '.o-expandable', Expandable );
+const navHeader = find( '.o-regs3k-navigation_header' );
+const navItems = find( '.o-regs3k-sections' );
 
-if ( 'serviceWorker' in navigator ) {
-  /* Delay registration until after the page has loaded, to ensure that our
```

Comment:
Should these comments move with the if statement to L:21?

Reply:
IDK, @contolini???